### ISSUE TITLE
Split panes not resized correctly when window size changes

### ISSUE THREAD

**Issue by joakimgunst** [nathancahill/Split.js#29]
1. Open http://nathancahill.github.io/Split.js/ in Chrome
2. Resize the browser window to about 800 pixels in width
3. Reload the page
4. Increase the width of the browser window

This causes empty white space to the right of the second split pane, which at least in my case is incorrect. The desired behavior would be to have the panes to increase in size to fill the container.

**Comment by PAEz** [comment]
When this error occurs its because the initial width for the column is not using calc(percent)-woteva but is setting the width to a hard value like 216px.  
This is happening in balancePairs when it calls the fit functions.  
These are only called if min or max are set.  
Gotta make those functions set the width to a percentage.

**Comment by PAEz** [comment]
The simplest solution (that seems to work, I dont know this code well enough to say for sure) is to put this....
````
for (i = pairs.length - 1; i >= 0; i--) {
                adjust.call(pairs[i],pairs[i].a[getBoundingClientRect]()[dimension])
            }
```
at the end of balancePairs.
You can see this working....
http://plnkr.co/edit/ndQdCPAiZ7t6lDA32m4i?p=preview
...yeah I know, shoulda PR, lazy, lazy, lazy

**Comment by pedrogit** [comment]
I had the same issue. PAEz's addition seems to fix it...

**Comment by nathancahill** [comment]
Ah, great @PAEz! I'll test this and get it merged in.

**Comment by jdanyow** [comment]
@PAEz's fix works for me too, except when there are three horizontal panes.  @PAEz's fix seems to help with the zoom issue too. :+1:

**Comment by PAEz** [comment]
@jdanyow any chance you could fork my plunker (you dont need to sign up or anything) and make an example of it failing?  I was going to have another look at this in a few days.

**Comment by zhy1stgg** [comment]
nice work..:thumbsup:

**Comment by narinearraul** [comment]
Hi all, just got this problem with my project. Has this fix been pushed to the lib yet?

**Comment by Wyzix33** [comment]
Problem not solved yet ...

**Comment by nathancahill** [comment]
Fixed in v1.2.0 by switching the balancePairs functionality to adjusting the minSize instead.
